src/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,58 @@
+import { SortBy } from '../types/SortBy';
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  sortByElement: SortBy;
+  reverseVisibleGoods: boolean;
+  changeFilter: (sortBy: SortBy) => void;
+  changeDirection: (isReverse: boolean) => void;
+}
+
+export const Buttons: React.FC<Props> = ({
+  sortByElement,
+  reverseVisibleGoods,
+  changeFilter = () => {},
+  changeDirection = () => {},
```

Comment:
```suggestion
  changeFilter,
  changeDirection,
```
These props are required, so you don't need to assign default values